python/modules/DeepTopProducer.py

```diff
@@ -97,9 +97,10 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
             tTagEffFileName = self.taggerWD + "/tTagEff_%(era)s.root"%{"era":self.era}
     
             if self.isFastSim:
-                sample = os.path.splitext(os.path.basename(inputFile.GetName()))[0]
+                sample = os.path.splitext(os.path.basename(inputFile.GetName()))[0].split("_split")[0]
             else:
-                sample = self.sampleName
+                # For QCD smear, using the orignal QCD
+                sample = self.sampleName.replace("Smear_", "")
```

Comment:
I guess other samples don't contain "Smear_" ...

Reply:
Yeah, in the current cfg, only QCD smear sample has "Smear". If other samples don't contain "Smear", the replace() won't affect the sampleName.